awx/main/dispatch/pool.py

```diff
@@ -445,6 +446,8 @@ def up(self):
             return super(AutoscalePool, self).up()
 
     def write(self, preferred_queue, body):
+        if 'guid' in body:
+            GuidMiddleware.set_guid(body['guid'])
```

Comment:
When the (parent) dispatcher process receives a message over the bus, set it in this thread so logs in the parent process include the guid